app/src/modules/content/routes/item.vue

```diff
@@ -700,6 +700,7 @@ function revert(values: Record<string, any>) {
 			:fields="fields"
 			:primary-key="internalPrimaryKey"
 			:validation-errors="validationErrors"
+			:current-version="currentVersion"
```

Comment:
To be consistent with other components (see below), I would recommend only using `version` here.

```suggestion
			:version="currentVersion"
```